python/Ganga/GPIDev/Adapters/IPostProcessor.py

```diff
@@ -85,8 +85,8 @@ def __str__(self):
 
     def append(self, value):
         self.addProcess(value)
-        self.process_objects = sorted(
-            self.process_objects, key=lambda process: process.order)
+        if hasattr(self.process_objects, 'order'):
+            self.process_objects = sorted(self.process_objects, key=lambda process: process.order)
```

Comment:
I couldn't tell you but the issue of the post-processors has come up before now and I'd rather look at it in a PR dedicated to changes of that system.